src/unit/format.js

```diff
@@ -0,0 +1,61 @@
+define([
+	"./get",
+	"../common/format-message"
+], function( unitGet, formatMessage ) {
+
+/**
+ * format( value, unit, options, cldr, globalize )
+ *
+ * @value [Number]
+ *
+ * @unit [String]:
+ *
+ * @options [Object]
+ * - form: [String] "long", "short" (default), or "narrow".
+ *
+ * FIXME
+ *
+ * Format units such as seconds, minutes, days, weeks, etc.
+ *
+ * OBS:
+ *
+ * Unit Sequences are not implemented.
+ * http://www.unicode.org/reports/tr35/tr35-35/tr35-general.html#Unit_Sequences
+ *
+ * Duration Unit (for composed time unit durations) is not implemented.
+ * http://www.unicode.org/reports/tr35/tr35-35/tr35-general.html#durationUnit
+ */
+return function( value, unit, options, pluralGenerator, cldr, globalize ) {
```

Comment:
Let's remove `globalize` as an argument. I know this comes from my previous branch. But, instead we should pass the plural formatter like https://github.com/jquery/globalize/blob/master/src/relative-time.js#L68. Because, it allows the runtime compiler to appropriately use a precompiled plural generator here.